src/xenia/base/memory.cc

```diff
@@ -27,14 +27,18 @@ void copy_128_aligned(void* dest, const void* src, size_t count) {
 void copy_and_swap_16_aligned(void* dest_ptr, const void* src_ptr,
                               size_t count) {
   assert_zero(reinterpret_cast<uintptr_t>(src_ptr) & 0x1);
+  assert_true((reinterpret_cast<uintptr_t>(src_ptr) & 0xF) ==
```

Comment:
On this - let's just make sure src_ptr and dest_ptr are aligned to a 16-byte boundary (in the aligned routines).
Any cases you run into where this is called with unaligned pointers you can just change to use the `*_unaligned` routines.

It's really confusing to make sure both pointers are unaligned the same way, and _very_ atypical for this to occur :)